tools/ports/__init__.py

```diff
@@ -400,11 +419,23 @@ def handle_use_port_error(arg, message):
 
 
 def handle_use_port_arg(settings, arg):
-  args = arg.split(':', 1)
-  name, options = args[0], None
+  if arg.find(':\\') == 1:  # detect windows path C:\path\port.py
+    drive, rest = arg[:2], arg[2:]  # drive=C:, rest=\path\port.py
+    args = rest.split(':', 1)
+    name, options = drive + args[0], None
```

Comment:
I don't love this code block.   For one thing it doesn't handle `c:/foo/bar.py` which is also valid on windows.

Can we simplify somehow?   Maybe:

```
# Ignore ':' in first or second char of string since we could be dealing with a windows driver separator
pos = arg.find(':', 2)
name = arg[:pos]
options = arg[pos:]

```

Reply:
Yeah me neither... stupid windows syntax :)

I will implement your suggestion!

Reply:
Implemented according to your suggestion (+ index fix :))